addons/website_blog/static/src/snippets/s_blog_posts/options.js

```diff
@@ -12,8 +12,13 @@ const dynamicSnippetBlogPostsOptions = dynamicSnippetOptions.extend({
         this._super.apply(this, arguments);
         this.modelNameFilter = 'blog.post';
         this.blogs = {};
+        this.tags = this.$target[0].dataset.filterByTagIds ? this.$target[0].dataset.filterByTagIds.split(",").map(id => parseInt(id)) : [];
     },
-
+    willStart: async function () {
```

Comment:
Prefer the form `async willStart() {` in master. It will be required with the new OWL-like system.